### ISSUE TITLE
Uncaught TypeError: Invalid Point: (NaN, 0)

### ISSUE THREAD

**Issue by kevindicke** [atom-minimap/minimap#608]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.11.6
**Thrown From**: [minimap](https://github.com/atom-minimap/minimap) package 4.28.2

### Stack Trace

Uncaught TypeError: Invalid Point: (NaN, 0)

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63

TypeError: Invalid Point: (NaN, 0)
    at Function.module.exports.Point.assertValid (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63:21)
    at DisplayLayer.translateScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:331:17)
    at DisplayLayer.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:389:18)
    at TextEditor.module.exports.TextEditor.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1720:38)
    at TextEditorComponent.module.exports.TextEditorComponent.pixelPositionForScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor-component.js:622:44)
    at atom-text-editor.TextEditorElement.pixelPositionForScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:304:29)
    at StableAdapter.computeScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:60:42)
    at StableAdapter.getScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:47:36)
    at Minimap.getTextEditorScaledScrollTop (/packages/minimap/lib/minimap.js:533:25)
    at /packages/minimap/lib/minimap-element.js:805:36)
    at requestAnimationFrame (/packages/minimap/lib/minimap-element.js:777:12)
```

### Commands

```
     -0:27.2.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling)
```

### Non-Core Packages

```
atom-beautify 0.30.1 
atom-runner 2.7.1 
busy-signal 1.4.3 
color-picker 2.2.5 
file-icons 2.1.7 
intentions 1.1.2 
language-babel 2.66.0 
linter 2.2.0 
linter-eslint 8.2.1 
linter-ui-default 1.6.1 
minimap 4.28.2 
pigments 0.39.1 
script 3.15.0 
swackets 0.26.0 
```

**Comment by brograhamer** [comment]
Receiving the same error, Windows 10, 1.9.1 Beta.

**Comment by kevteljeur** [comment]
Receiving the same error, Mac OS, 1.19.0-beta2 on splitting current tab to new pane.

**Comment by steronydh** [comment]
Same error, Windows 10 64-bit, Atom 1.19.0-beta2 on splitting tab.

**Comment by wyqydsyq** [comment]
Also getting this on OpenSUSE Tumbleweed, `Atom 1.19.0-beta-2` upon double-clicking a token (to highlight other usages)

**Comment by kevteljeur** [comment]
Here's something. The error only happens when plugins are enabled. If I use minimap by itself and with auto-hide plugin, there's no problem. It appears to require a restart of Atom for that. So, that's a problem with co-ordinates getting passed from the buffer display to minimap?

Also, I suspect that this project isn't being maintained at the moment, so we may have to fix it ourselves.

**Comment by ahouse101** [comment]
This still happens to me on backspace/delete in some files, very frustrating.

**Comment by trevordunn** [comment]
Just saw this today. Can't split up/down/left/right.

**Atom**: 1.19.0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.11.6
**Thrown From**: [minimap](https://github.com/atom-minimap/minimap) package 4.29.3


### Stack Trace

Uncaught TypeError: Invalid Point: (NaN, 0)

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63

TypeError: Invalid Point: (NaN, 0)
    at Function.module.exports.Point.assertValid (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63:21)
    at DisplayLayer.translateScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:333:17)
    at DisplayLayer.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:391:18)
    at TextEditor.module.exports.TextEditor.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1793:38)
    at HTMLElement.pixelPositionForScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:242:38)
    at StableAdapter.computeScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:60:42)
    at StableAdapter.getScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:51:17)
    at Minimap.getTextEditorScrollRatio (/packages/minimap/lib/minimap.js:598:25)
    at Minimap.getCapedTextEditorScrollRatio (/packages/minimap/lib/minimap.js:610:29)
    at Minimap.getScrollTopFromEditor (/packages/minimap/lib/minimap.js:937:12)
    at Minimap.getScrollTop (/packages/minimap/lib/minimap.js:887:14)
    at Minimap.getLastVisibleScreenRow (/packages/minimap/lib/minimap.js:865:13)
    at Minimap.emitRangeChanges (/packages/minimap/lib/mixins/decoration-management.js:503:38)
    at Minimap.emitDecorationChanges (/packages/minimap/lib/mixins/decoration-management.js:488:10)
    at Minimap.decorateMarker (/packages/minimap/lib/mixins/decoration-management.js:418:10)
    at MinimapCursorLineBinding.handleMarker (/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:21:37)
    at /packages/minimap-cursorline/lib/minimap-cursorline-binding.js:15:12
    at TextEditor.module.exports.TextEditor.observeCursors (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:744:15)
    at /packages/minimap-cursorline/lib/minimap-cursorline-binding.js:14:40)
    at /packages/minimap-cursorline/lib/minimap-cursorline.js:40:23
    at onDidCreateMinimap (/packages/minimap/lib/main.js:380:51)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at subscriptions.add.atom.workspace.observeTextEditors (/packages/minimap/lib/main.js:393:20)
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:721:22
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:594:34
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:595:19
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.didAddPaneItem (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:270:24)
    at PaneContainer.didAddPane (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:242:18)
    at Pane.module.exports.Pane.setContainer (/Applications/Atom.app/Contents/Resources/app/src/pane.js:162:32)
    at PaneAxis.module.exports.PaneAxis.addChild (/Applications/Atom.app/Contents/Resources/app/src/pane-axis.js:154:19)
    at PaneAxis.module.exports.PaneAxis.insertChildBefore (/Applications/Atom.app/Contents/Resources/app/src/pane-axis.js:219:25)
    at Pane.module.exports.Pane.split (/Applications/Atom.app/Contents/Resources/app/src/pane.js:1001:29)
    at Pane.module.exports.Pane.splitLeft (/Applications/Atom.app/Contents/Resources/app/src/pane.js:960:25)
    at HTMLElement.pane:split-left-and-copy-active-item (/Applications/Atom.app/Contents/Resources/app/src/register-default-commands.js:264:38)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1338:34)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:334:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -2:34.1.0 pane:split-right-and-copy-active-item (input.hidden-input)
     -1:24.5.0 pane:split-left-and-copy-active-item (input.hidden-input)
     -0:58.3.0 pane:split-right-and-copy-active-item (input.hidden-input)
```

### Non-Core Packages

```
api-workbench 0.8.46 
atom-beautify 0.30.4 
atom-monokai 0.10.9 
atom-typescript 11.0.6 
busy-signal 1.4.3 
chary-tree-view 0.2.13 
color-picker 2.2.5 
docblockr 0.11.0 
editorconfig 2.2.2 
file-types 0.5.5 
haskell-grammar 0.4.0 
highlight-selected 0.13.1 
intentions 1.1.5 
keyboard-scroll 0.7.0 
language-actionscript3 0.3.0 
language-babel 2.69.0 
language-htaccess 0.1.1 
language-json-comments 0.16.3 
language-nginx 0.8.0 
language-typescript-grammars-only 1.6.0 
linter 2.2.0 
linter-ui-default 1.6.3 
merge-conflicts 1.4.5 
minimap 4.29.3 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.2 
minimap-highlight-selected 4.6.1 
monokai-extended 0.3.0 
multi-wrap-guide 0.29.1 
node-debugger 1.10.1 
nord-atom-ui 0.11.0 
nuclide 0.243.0 
pane-manager 1.0.1 
pigments 0.40.1 
platformio-ide-terminal 2.5.5 
project-manager 3.3.5 
sync-settings 0.8.2 
terminal-plus 0.14.5 
tree-ignore 0.6.2 
```

**Comment by pjenx** [comment]
+1 
Experienced this behavior only after updating to 1.19.0.
Split screen action pops this error which links to this closed issue.
When restarting atom, the desired split screen is in effect.

**Comment by lucasdf** [comment]
+1 having this issue with 1.19 while using [split-diff](https://github.com/mupchrch/split-diff/issues)

**Comment by mdotwills** [comment]
1. Right click open code tab and choose split option.

**Atom**: 1.19.0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [minimap](https://github.com/atom-minimap/minimap) package 4.29.6


### Stack Trace

Uncaught TypeError: Invalid Point: (NaN, 0)

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63

TypeError: Invalid Point: (NaN, 0)
    at Function.module.exports.Point.assertValid (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63:21)
    at DisplayLayer.translateScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:333:17)
    at DisplayLayer.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:391:18)
    at TextEditor.module.exports.TextEditor.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1793:38)
    at HTMLElement.pixelPositionForScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:242:38)
    at StableAdapter.computeScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:60:42)
    at StableAdapter.getScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:51:17)
    at Minimap.getTextEditorScrollRatio (/packages/minimap/lib/minimap.js:598:25)
    at Minimap.getCapedTextEditorScrollRatio (/packages/minimap/lib/minimap.js:610:29)
    at Minimap.getScrollTopFromEditor (/packages/minimap/lib/minimap.js:937:12)
    at Minimap.getScrollTop (/packages/minimap/lib/minimap.js:887:14)
    at Minimap.getLastVisibleScreenRow (/packages/minimap/lib/minimap.js:865:13)
    at Minimap.emitRangeChanges (/packages/minimap/lib/mixins/decoration-management.js:504:38)
    at Minimap.emitDecorationChanges (/packages/minimap/lib/mixins/decoration-management.js:489:10)
    at Minimap.decorateMarker (/packages/minimap/lib/mixins/decoration-management.js:422:10)
    at MinimapCursorLineBinding.handleMarker (/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:21:37)
    at /packages/minimap-cursorline/lib/minimap-cursorline-binding.js:15:12
    at TextEditor.module.exports.TextEditor.observeCursors (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:744:15)
    at /packages/minimap-cursorline/lib/minimap-cursorline-binding.js:14:40)
    at /packages/minimap-cursorline/lib/minimap-cursorline.js:40:23
    at onDidCreateMinimap (/packages/minimap/lib/main.js:380:51)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at subscriptions.add.atom.workspace.observeTextEditors (/packages/minimap/lib/main.js:393:20)
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:721:22
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:594:34
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:595:19
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
```

### Commands

```
     -1:38.9.0 editor:newline (input.hidden-input)
     -1:38.8.0 snippets:next-tab-stop (input.hidden-input)
     -1:38.8.0 snippets:expand (input.hidden-input)
     -1:38.8.0 editor:indent (input.hidden-input)
     -1:38.5.0 core:backspace (input.hidden-input)
     -1:37.9.0 snippets:next-tab-stop (input.hidden-input)
     -1:37.9.0 snippets:expand (input.hidden-input)
     -1:37.9.0 editor:indent (input.hidden-input)
     -1:35 core:backspace (input.hidden-input)
     -1:30.2.0 core:save (input.hidden-input)
     -1:13.7.0 tabs:split-down (div.title)
```

### Non-Core Packages

```
atom-panda-syntax 0.18.0 
highlight-selected 0.13.1 
language-javascript-jsx 0.3.7 
minimap 4.29.6 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-pigments 0.2.2 
minimap-selection 4.5.0 
path-copy 0.14.0 
pigments 0.40.2 
tree-view-copy-relative-path 1.2.0 
```

**Comment by pjenx** [comment]
Just to clarify:

This error does _not_ occur when using the Atom commands (e.g. Pane: Split Left).

But like [lucasdf](https://github.com/atom-minimap/minimap/issues/608#issuecomment-321439702), it occurs when using [split-diff](https://github.com/mupchrch/split-diff/issues) (Split-Diff: Toggle) 
and like [mdotwills](https://github.com/atom-minimap/minimap/issues/608#issuecomment-321455152), it occurs on right-click tab, choose split menu option.

This error is thrown, the split does not occur, and then on restart the split is in effect. Behavior started after the Atom 1.19.0 update, before and after updating minimap to 4.29.6

**Comment by johnhaley81** [comment]
It happened to me when using the the Atom command. Restarting Atom fixed it. I do not have `split-diff` installed, nor do I use the mouse to split windows.

**Comment by mdotwills** [comment]
@pjenx that is right, same behaviour here. @johnhaley81 restarting tom has not fixed this issue.

This is a closed issue, who do we ping to look at opening it or do we have to open another issue?

**Comment by bazzilic** [comment]
Same issue here. Happened when I tried to split the panes using the top menu.

**Comment by gluons** [comment]
Atom error bring me here. Error happen when use `split-diff` or use Atom split menu.

**Comment by binarykitchen** [comment]
It's frustrating and cannot use atom anymore at all :/

**Comment by jacobmischka** [comment]
It's going to be fixed in 1.19.1, we'll just have to live without the minimap for a few more days until then.

**Comment by johnyf** [comment]
This problem isn't fixed in Atom 1.20.0 beta, please reopen this issue.

Error:


[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.20.0-beta1 x64
**Electron**: 1.6.9
**OS**: Debian GNU/Linux
**Thrown From**: [minimap](https://github.com/atom-minimap/minimap) package 4.29.6


### Stack Trace

Uncaught TypeError: Invalid Point: (NaN, 0)

```
At /usr/share/atom-beta/resources/app/node_modules/text-buffer/lib/point.js:63

TypeError: Invalid Point: (NaN, 0)
    at Function.module.exports.Point.assertValid (/usr/share/atom-beta/resources/app/node_modules/text-buffer/lib/point.js:63:21)
    at DisplayLayer.translateScreenPosition (/usr/share/atom-beta/resources/app/node_modules/text-buffer/lib/display-layer.js:333:17)
    at DisplayLayer.clipScreenPosition (/usr/share/atom-beta/resources/app/node_modules/text-buffer/lib/display-layer.js:391:18)
    at TextEditor.module.exports.TextEditor.clipScreenPosition (/usr/share/atom-beta/resources/app/src/text-editor.js:1793:38)
    at HTMLElement.pixelPositionForScreenPosition (/usr/share/atom/resources/app/src/text-editor-element.js:242:38)
    at StableAdapter.computeScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:60:42)
    at StableAdapter.getScrollTop (/packages/minimap/lib/adapters/stable-adapter.js:51:17)
    at Minimap.getTextEditorScrollRatio (/packages/minimap/lib/minimap.js:598:25)
    at Minimap.getCapedTextEditorScrollRatio (/packages/minimap/lib/minimap.js:610:29)
    at Minimap.getScrollTopFromEditor (/packages/minimap/lib/minimap.js:937:12)
    at Minimap.getScrollTop (/packages/minimap/lib/minimap.js:887:14)
    at Minimap.getLastVisibleScreenRow (/packages/minimap/lib/minimap.js:865:13)
    at Minimap.emitRangeChanges (/packages/minimap/lib/mixins/decoration-management.js:504:38)
    at Minimap.emitDecorationChanges (/packages/minimap/lib/mixins/decoration-management.js:489:10)
    at Minimap.decorateMarker (/packages/minimap/lib/mixins/decoration-management.js:422:10)
    at MinimapCursorLineBinding.handleMarker (/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:21:37)
    at /packages/minimap-cursorline/lib/minimap-cursorline-binding.js:15:12
    at TextEditor.module.exports.TextEditor.observeCursors (/usr/share/atom-beta/resources/app/src/text-editor.js:744:15)
    at /packages/minimap-cursorline/lib/minimap-cursorline-binding.js:14:40)
    at /packages/minimap-cursorline/lib/minimap-cursorline.js:40:23
    at onDidCreateMinimap (/packages/minimap/lib/main.js:380:51)
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom-beta/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom-beta/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at subscriptions.add.atom.workspace.observeTextEditors (/packages/minimap/lib/main.js:393:20)
    at /usr/share/atom-beta/resources/app/src/workspace.js:729:22
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom-beta/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom-beta/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at /usr/share/atom-beta/resources/app/src/workspace.js:583:34
    at /usr/share/atom-beta/resources/app/src/workspace.js:584:19
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom-beta/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom-beta/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.didAddPaneItem (/usr/share/atom-beta/resources/app/src/pane-container.js:270:24)
    at PaneContainer.didAddPane (/usr/share/atom-beta/resources/app/src/pane-container.js:242:18)
    at Pane.module.exports.Pane.setContainer (/usr/share/atom-beta/resources/app/src/pane.js:162:32)
    at PaneAxis.module.exports.PaneAxis.addChild (/usr/share/atom-beta/resources/app/src/pane-axis.js:154:19)
    at PaneAxis.module.exports.PaneAxis.insertChildAfter (/usr/share/atom-beta/resources/app/src/pane-axis.js:225:25)
    at Pane.module.exports.Pane.split (/usr/share/atom-beta/resources/app/src/pane.js:1012:29)
    at Pane.module.exports.Pane.splitRight (/usr/share/atom-beta/resources/app/src/pane.js:972:25)
    at HTMLElement.pane:split-right-and-copy-active-item (/usr/share/atom-beta/resources/app/src/register-default-commands.js:269:38)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom-beta/resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/usr/share/atom-beta/resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/share/atom-beta/resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/usr/share/atom-beta/resources/app/src/atom-environment.js:1344:34)
    at EventEmitter.outerCallback (/usr/share/atom-beta/resources/app/src/application-delegate.js:348:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -0:32.2.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
     -0:21 pane:split-right-and-copy-active-item (div.line.cursor-line.highlight-line)
```

**Comment by pjvalla** [comment]
This issue really needs to be re-opened.  I am having the identical issue that @johnyf is having.

**Comment by aadamsx** [comment]
Switching now back to VSCode due to this issue -- can't split screens anymore!

**Comment by johnyf** [comment]
@aadamsx uninstalling `minimap` addressed the issue in my case. I also experienced lags when editing and scrolling large files that were in focus in multiple tabs. Removing `minimap` addressed those lags too, which was a little unexpected.

**Comment by dflock** [comment]
This has started happening to me with Atom 1.20.0-beta2. This issue needs to be re-opened, it's not fixed.

**Comment by ahyanarizky** [comment]
got same error in 

Atom    : 1.19.0
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0
OS  : macOS Sierra 10.12.6

**Comment by greschd** [comment]
For me, disabling only ``minimap-cursorline`` and restarting Atom fixes the issue.

**Comment by jacobmischka** [comment]
Can confirm that's working for me too, thanks @greschd!

**Comment by lucasdf** [comment]
I have not get this problem again since I upgraded to 1.19.1

**Comment by jacobmischka** [comment]
I seem to still have it with minimap-cursorline enabled on 1.19.1, unfortunately.

**Comment by WLyKan** [comment]
upgrade to 1.19.2 just now，and still have it

**Comment by eduardobc** [comment]
S.O:
macOS Sierra 10.12.6 (16G29)
Uncaught TypeError:
`Invalid Point: (NaN, 0)`
in:
`/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63`
TypeError: 
```
Invalid Point: (NaN, 0)
    at Function.module.exports.Point.assertValid (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63:21)
    at DisplayLayer.translateScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:333:17)
    at DisplayLayer.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:391:18)
    at TextEditor.module.exports.TextEditor.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1794:38)
    at HTMLElement.pixelPositionForScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:242:38)
    at StableAdapter.computeScrollTop (/Users/Eduado/.atom/packages/minimap/lib/adapters/stable-adapter.js:60:42)
    at StableAdapter.getScrollTop (/Users/Eduado/.atom/packages/minimap/lib/adapters/stable-adapter.js:51:17)
    at Minimap.getTextEditorScrollRatio (/Users/Eduado/.atom/packages/minimap/lib/minimap.js:598:25)
    at Minimap.getCapedTextEditorScrollRatio (/Users/Eduado/.atom/packages/minimap/lib/minimap.js:610:29)
    at Minimap.getScrollTopFromEditor (/Users/Eduado/.atom/packages/minimap/lib/minimap.js:937:12)
    at Minimap.getScrollTop (/Users/Eduado/.atom/packages/minimap/lib/minimap.js:887:14)
    at Minimap.getLastVisibleScreenRow (/Users/Eduado/.atom/packages/minimap/lib/minimap.js:865:13)
    at Minimap.emitRangeChanges (/Users/Eduado/.atom/packages/minimap/lib/mixins/decoration-management.js:504:38)
    at Minimap.emitDecorationChanges (/Users/Eduado/.atom/packages/minimap/lib/mixins/decoration-management.js:489:10)
    at Minimap.decorateMarker (/Users/Eduado/.atom/packages/minimap/lib/mixins/decoration-management.js:422:10)
    at MinimapHighlightSelected.markerCreated (/Users/Eduado/.atom/packages/minimap-highlight-selected/lib/minimap-highlight-selected.coffee:53:26)
    at MinimapHighlightSelected.markerCreated (/Users/Eduado/.atom/packages/minimap-highlight-selected/lib/minimap-highlight-selected.coffee:1:1)
    at /Users/Eduado/.atom/packages/minimap-highlight-selected/lib/minimap-highlight-selected.coffee:39:62
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at /Users/Eduado/.atom/packages/highlight-selected/lib/highlighted-area-view.coffee:163:20
    at ForwardsSingleLine.iterate (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/match-iterator.js:94:17)
    at TextBuffer.module.exports.TextBuffer.scanInRange (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:1033:29)
    at TextEditor.module.exports.TextEditor.scanInBufferRange (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:2737:32)
    at HighlightedAreaView.module.exports.HighlightedAreaView.highlightSelectionInEditor (/Users/Eduado/.atom/packages/highlight-selected/lib/highlighted-area-view.coffee:151:12)
    at /Users/Eduado/.atom/packages/highlight-selected/lib/highlighted-area-view.coffee:136:10
    at Array.forEach (native)
    at HighlightedAreaView.module.exports.HighlightedAreaView.handleSelection (/Users/Eduado/.atom/packages/highlight-selected/lib/highlighted-area-view.coffee:135:27)
    at HighlightedAreaView.handleSelection (/Users/Eduado/.atom/packages/highlight-selected/lib/highlighted-area-view.coffee:1:1)
    at /Users/Eduado/.atom/packages/highlight-selected/lib/highlighted-area-view.coffee:62:8
```

**Comment by ineil** [comment]
Same issue - macOS Sierra 10.12.6, Atom 1.19.2 release,

```
TypeError: Invalid Point: (NaN, 0)
    at Function.module.exports.Point.assertValid (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/point.js:63:21)
    at DisplayLayer.translateScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:333:17)
    at DisplayLayer.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/display-layer.js:391:18)
    at TextEditor.module.exports.TextEditor.clipScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1794:38)
    at HTMLElement.pixelPositionForScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:242:38)
    at StableAdapter.computeScrollTop (/Users/neil/.atom/packages/minimap/lib/adapters/stable-adapter.js:60:42)
    at StableAdapter.getScrollTop (/Users/neil/.atom/packages/minimap/lib/adapters/stable-adapter.js:51:17)
    at Minimap.getTextEditorScrollRatio (/Users/neil/.atom/packages/minimap/lib/minimap.js:598:25)
    at Minimap.getCapedTextEditorScrollRatio (/Users/neil/.atom/packages/minimap/lib/minimap.js:610:29)
    at Minimap.getScrollTopFromEditor (/Users/neil/.atom/packages/minimap/lib/minimap.js:937:12)
    at Minimap.getScrollTop (/Users/neil/.atom/packages/minimap/lib/minimap.js:887:14)
    at Minimap.getLastVisibleScreenRow (/Users/neil/.atom/packages/minimap/lib/minimap.js:865:13)
    at Minimap.emitRangeChanges (/Users/neil/.atom/packages/minimap/lib/mixins/decoration-management.js:504:38)
    at Minimap.emitDecorationChanges (/Users/neil/.atom/packages/minimap/lib/mixins/decoration-management.js:489:10)
    at Minimap.decorateMarker (/Users/neil/.atom/packages/minimap/lib/mixins/decoration-management.js:422:10)
    at MinimapCursorLineBinding.handleMarker (/Users/neil/.atom/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:21:37)
    at /Users/neil/.atom/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:15:12
    at TextEditor.module.exports.TextEditor.observeCursors (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:745:15)
    at new MinimapCursorLineBinding (/Users/neil/.atom/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:14:40)
    at /Users/neil/.atom/packages/minimap-cursorline/lib/minimap-cursorline.js:40:23
    at onDidCreateMinimap (/Users/neil/.atom/packages/minimap/lib/main.js:380:51)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at subscriptions.add.atom.workspace.observeTextEditors (/Users/neil/.atom/packages/minimap/lib/main.js:393:20)
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:721:22
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:594:34
    at /Applications/Atom.app/Contents/Resources/app/src/workspace.js:595:19
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.didAddPaneItem (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:270:24)
    at PaneContainer.didAddPane (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:242:18)
    at Pane.module.exports.Pane.setContainer (/Applications/Atom.app/Contents/Resources/app/src/pane.js:162:32)
    at PaneAxis.module.exports.PaneAxis.addChild (/Applications/Atom.app/Contents/Resources/app/src/pane-axis.js:154:19)
    at PaneAxis.module.exports.PaneAxis.insertChildBefore (/Applications/Atom.app/Contents/Resources/app/src/pane-axis.js:219:25)
    at Pane.module.exports.Pane.split (/Applications/Atom.app/Contents/Resources/app/src/pane.js:1001:29)
    at Pane.module.exports.Pane.splitLeft (/Applications/Atom.app/Contents/Resources/app/src/pane.js:960:25)
    at HTMLElement.pane:split-left-and-copy-active-item (/Applications/Atom.app/Contents/Resources/app/src/register-default-commands.js:264:38)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1338:34)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:334:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
The error was thrown from the minimap package. This issue has already been reported.
```

**Comment by amyrebecca** [comment]
Same behavior, Atom 1.19.2, MacOS 10.12.5, Minimap 4.29.6

**Comment by pjenx** [comment]
I tried to do some debugging on this by disabling/enabling packages and found I no longer get the error with the minimap-cursorline package disabled.

There's actually a little more to it... once I am getting the error, simply disabling minimap-cursorline doesn't fix it.  I need to disable both minimap and minimap-cursorline, then re-enable minimap but leave minimap-cursorline disabled. That seems to reset it and the error no longer occurs.

Granted this solution doesn't stem from an understanding of the problem and is not a real fix, but working is good. Hopefully it is stable and the error doesn't recur, but it seems to work.

**Comment by Tokimon** [comment]
I have the same issue when I right-click the tab->"Split right".
It persists even if I have disabled `minimap-cursorline`.

Obviously something has been changed that modules depending on `minimap` are using thus giving them the `NaN` point.

**Modules I use, causing this problem:**
`minimap-cursorline`
`minimap-selection`

**Comment by UziTech** [comment]
Is anyone still having this issue?
